docs/server/source/simple-network-setup.md

```diff
@@ -0,0 +1,209 @@
+
+```json
+{
+  "address": "5943A9EF6285791A504918E1D117BC7F6A615C98",
+  "pub_key": {
+    "type": "AC26791624DE60",
+    "value": "W3tqeMCU3d4SHDKqrwQWTahTW/wpieIAKZQxUxLv6rI="
+  },
+  "last_height": 0,
+  "last_round": 0,
+  "last_step": 0,
+  "priv_key": {
+    "type": "954568A3288910",
+    "value": "3sv9aExgME6MMjx0JoKVy7KtED8PBiPcyAgsYmVizslbe2p4wJTd3hIcMqqvBBZNqFNb/CmJ4gAplDFTEu/qsg=="
+  }
+}
+```
+
+To extract your `node_id`, run the command:
+
+```
+tendermint show_node_id
+```
+
+Share with all other Members your `node_id`, `pub_key.value`, and the host name or IP of your Node.
+
+**Important note on security: each Member should verify the data they receive from the other Members.**
+
+## Coordinator: Initialize the Network
+At this point the Coordinator should has received the data from all the Members, and should combine them in the `.tendermint/config/genesis.json` file:
+
+```json
+{
+  "genesis_time": "0001-01-01T00:00:00Z",
+  "chain_id": "test-chain-la6HSr",
+  "validators": [
+    {
+      "pub_key": {
+        "type": "AC26791624DE60",
+        "value": "<Member 1 public key"
+      },
+      "power": 10,
+      "name": "<Member 1 name>"
+    },
+    {
+     "pub_key": {
+       "type": "AC26791624DE60",
+       "value": "<Member 2 public key>"
+     },
+     "power": 10,
+      "name": "<Member 2 name>"
+    },
+    {
+     "...": { },
+    },
+    {
+     "pub_key": {
+       "type": "AC26791624DE60",
+       "value": "<Member N public key>"
+     },
+     "power": 10,
+      "name": "<Member N name>"
+    }
+  ],
+  "app_hash": ""
+}
+```
+
+The new `genesis.json` file contains the data that describes the Network.
+
+At this point, the Coordinator must share the new `genesis.json` file with all Members.
+
+## Member: Connect to the other Members
+At this point the Member should have received the `genesis.json` file.
+
+**Important note on security: each Member should verify that the `genesis.json` file contains the correct public keys.**
+
+The Member must copy the `genesis.json` file in the local `.tendermint/config` directory. Every Member now shares the same `chain_id`, `genesis_time`, used to identify the Network, and the same list of `validators`.
+
+The Member must edit the `.tendermint/config/config.toml` file and make the following changes:
+
+```
+...
+
+create_empty_blocks = false
+...
+
+persistent_peers = "<Member 1 node id>@<Member 1 hostname>:46656,\
+<Member 2 node id>@<Member 2 hostname>:46656,\
+<Member N node id>@<Member N hostname>:46656,"
+```
+
+Now the Member must run two commands:
+- `bigchaindb start`
+- `tendermint node`
+
+The commands will run on the foreground, you might want to use `nohup`, `tmux`, or `screen` to keep them running after you logout. The recommended approach is to create startup scripts for both BigchainDB and Tendermint, so they will start right after the boot of the operating system.
```

Comment:
Maybe we could offer a little more help here, to be in line with the level of help so far. Maybe give some _example_ commands that will run MongoDB, BigchainDB and Tendermint in the background, e.g.
```
sudo mongod --fork --logpath mongodb.log
sudo nohup tendermint node 2>&1 > tendermint.log &
sudo nohup bigchaindb start 2>&1 > bigchaindb.log &
```
and then mention "The recommended approach..." afterwards.

Reply:
I'd like to add the two `systemd` unit files here, if you don't mind I'll address this in another PR :)

Reply:
So the idea would be that they change the startup behavior of the VM, so that on reboot, MongoDB, BigchainDB and Tendermint will all start? That should be explained.